index.js

```diff
@@ -3,30 +3,28 @@ var path = require('path');
 var pathExists = require('path-exists');
 var Promise = require('pinkie-promise');
 
-function splitPath(x) {
-	return path.resolve(x || '').split(path.sep);
-}
-
-function join(parts, filename) {
-	return path.resolve(parts.join(path.sep) + path.sep, filename);
+function rootDir(dir) {
+	var sepIndex = dir.indexOf(path.sep);
+	return sepIndex >= 0 ? dir.substring(0, sepIndex + 1) : dir;
```

Comment:
Updated to use `path.parse()`. Tested Node 0.12, 4, and 6, which work. And Node 0.10, which does not.